src/nspawn/nspawn-mount.h

```diff
@@ -23,6 +23,14 @@
 
 #include "cgroup-util.h"
 
+typedef enum MountSettingsMask {
+        MOUNT_FATAL             = 1 << 0,
+        MOUNT_IN_USERNS         = 1 << 1,
+        MOUNT_USE_USERNS        = 1 << 3,
```

Comment:
if we have these nice flags, I think we should also cease the chance and document them in a nice way, i.e. add some `/* */` explanation here to the end of relevant flags. I figure most flags are pretty much self-explanatory, but I think the namespacing-related options could really use a brief explanation, i.e. what the difference between "IN_USERNS" and "USE_USERNS" actually is...